### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by mayank2245**

### CODE REVIEW

File: src/components/appbar/DBAppBar.js [hunk 1]
```diff
@@ -84,19 +84,17 @@ const DBAppBar = ({
...
-            icon={editIcon}
-            _pressed={{
-              backgroundColor: theme.colors.secondary[200],
-            }}
-            onPress={() => navigation.navigate('EditLocation')}
-          />
-        </HStack>
+        <TouchableOpacity
+          activeOpacity={0.4}
+          onPress={() => navigation.navigate('EditLocation')}>
```
Comment (VishalKrHangingPanda) [inline src/components/appbar/DBAppBar.js:89]: use Const for onPress
and call that const here

File: src/screens/account/Account.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+          _web={{
+            shadow: 2,
+            borderWidth: 0,
+          }}
+          _light={{
+            backgroundColor: 'gray.50',
+          }}>
+          <FlatList
+            data={data1}
+            renderItem={({item}) => (
```
Comment (VishalKrHangingPanda) [inline src/screens/account/Account.js:314]: render this element in separate const

Comment (mayank2245) [reply src/screens/account/Account.js:314]: okay sir i got it 👍

File: src/components/appbar/DBAppBar.js (hunk 1 above)
Comment (mayank2245) [reply src/components/appbar/DBAppBar.js:89]: okay i am changing it sir

**Pull request merged**